<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSIMExportConnectionGraphDialog</class>
 <widget class="QDialog" name="MSIMExportConnectionGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export/print connection graph...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabPrint">
      <attribute name="title">
       <string>Print</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayoutPrint">
       <item>
        <widget class="QWidget" name="widgetPrintOptions" native="true">
         <layout class="QVBoxLayout" name="verticalLayoutPrintOptions">
          <item>
           <widget class="QPushButton" name="pushButtonPageSetup">
            <property name="text">
             <string>Page setup...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonPrint">
            <property name="text">
             <string>Print...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/MasterSimulator.qrc">
              <normaloff>:/gfx/actions/24x24/print.png</normaloff>:/gfx/actions/24x24/print.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Vertical alignment:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widgetVAlignment" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QToolButton" name="toolButtonAlignTop">
               <property name="icon">
                <iconset resource="../resources/MasterSimulator.qrc">
                 <normaloff>:/gfx/actions/24x24/image_top_aligned.png</normaloff>:/gfx/actions/24x24/image_top_aligned.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButtonAlignVCenter">
               <property name="icon">
                <iconset resource="../resources/MasterSimulator.qrc">
                 <normaloff>:/gfx/actions/24x24/image_vcenter_aligned.png</normaloff>:/gfx/actions/24x24/image_vcenter_aligned.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButtonAlignBottom">
               <property name="icon">
                <iconset resource="../resources/MasterSimulator.qrc">
                 <normaloff>:/gfx/actions/24x24/image_bottom_aligned.png</normaloff>:/gfx/actions/24x24/image_bottom_aligned.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Horizontal alignment:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widgetVAlignment_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QToolButton" name="toolButtonAlignLeft">
               <property name="icon">
                <iconset resource="../resources/MasterSimulator.qrc">
                 <normaloff>:/gfx/actions/24x24/image_left_aligned.png</normaloff>:/gfx/actions/24x24/image_left_aligned.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButtonAlignHCenter">
               <property name="icon">
                <iconset resource="../resources/MasterSimulator.qrc">
                 <normaloff>:/gfx/actions/24x24/image_hcenter_aligned.png</normaloff>:/gfx/actions/24x24/image_hcenter_aligned.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButtonAlignRight">
               <property name="icon">
                <iconset resource="../resources/MasterSimulator.qrc">
                 <normaloff>:/gfx/actions/24x24/image_right_aligned.png</normaloff>:/gfx/actions/24x24/image_right_aligned.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Scale factor [%]:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxScaleFactor">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabExport">
      <attribute name="title">
       <string>Export</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,1">
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonExportImage">
         <property name="text">
          <string>Export...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>14</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QRadioButton" name="radioButtonBitmap">
         <property name="text">
          <string>Bitmap</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QRadioButton" name="radioButtonVector">
         <property name="text">
          <string>Vector graphics</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelW">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Max. width:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelH">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Max. height:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="spinBoxWidth">
           <property name="minimumSize">
            <size>
             <width>89</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>800</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxHeight">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>600</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2" rowspan="5">
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>759</width>
            <height>718</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutScrollArea">
           <item>
            <widget class="QLabel" name="labelPreview">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background:white</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/MasterSimulator.qrc"/>
 </resources>
 <connections/>
</ui>
